samples/sensor/mcp9808/sample.yaml

```diff
@@ -4,4 +4,4 @@ tests:
   sample.sensor.mcp9808:
     harness: sensor
     tags: sensors
-    depends_on: i2c
+    depends_on: i2c mcp9808
```

Comment:
There aren't any boards that set `mcp9808` as supported, so this change will cause the sample to never build in CI. Can you either revert this change or add `mcp9808` to the supported list for an appropriate board?

Reply:
I don't know any already supported board that has that sensor, so I'll revert that change, as suggested.